### DISCUSSION TITLE
How can I extend a extension inside starter kit?

### DISCUSSION THREAD

**Discussion by eliabexp** [ueberdosis/tiptap#4972] [General Questions]
The title is auto explained

**Comment by abecirovic3** [comment]
If I understand the question correctly, you can configure the starter kit extension to disable the extension in question. After that, you can extend the extension and include your extension separately. For example:
```
const editor = new Editor({
  extensions: [
    StarterKit.configure({
      // Disable an included extension
      heading: false,
    }),
   CustomHeading // custom heading is your extended heading extension
  ],
})


```
